_layouts/post.html

```diff
@@ -37,8 +49,20 @@ <h1 class="title page-title">{{page.title}}</h1>
     <div class="generated-content">
         {{content}}
     </div>
+
+<hr class="divider" />
 </article>
 
+
+<div class = "social-buttons" style="text-align: center">
```

Comment:
Instead of using `style="text-align: center"`, let's bundle that in to the `.social-buttons` class.